src/rules/utils.ts

```diff
@@ -633,6 +633,25 @@ export const isHook = (
   node.callee.type === AST_NODE_TYPES.Identifier &&
   HookName.hasOwnProperty(node.callee.name);
 
+export const getTestCallExpressionsFromDeclaredVariables = <
+  R extends JestFunctionCallExpression<TestCaseName>
```

Comment:
I don't think this generic is actually adding much  - let's keep things simple, and just stick with `JestFunctionCallExpression<TestCaseName>` if possible :)